config/install.yaml

```diff
@@ -842,6 +851,9 @@ spec:
     - jsonPath: .status.replicas
       name: Current
       type: string
+    - jsonPath: .status.readyReplicas
+      name: Ready
+      type: string
```

Comment:
seems this is getting updated since Numaflow CRDs have been updated in `numaflow:stable`